SMPyBandits/Arms/RestedRottingArm.py

```diff
@@ -0,0 +1,69 @@
+"""
+author: Julien Seznec
+Rested rotting arm, i.e. arms with mean value which decay at each pull
+"""
+
+from . import Arm, Bernoulli, Binomial, UnboundedExponential, UnboundedGaussian, Constant, UnboundedPoisson
+
+
+class RestedRottingArm(Arm):
+    def __init__(self, decayingFunction, staticArm):
```

Comment:
Could you add a doctring to each class, at least a one-liner saying something like
```python
    """ Rested rotting version of an exponentially distributed arm. """
```

Reply:
I will